x11-wm/qtile/qtile-0.22.1-r1.ebuild

```
@@ -37,7 +37,7 @@ RDEPEND="
 		media-sound/pulseaudio
 	)
 	wayland? (
-		dev-python/pywlroots[${PYTHON_USEDEP}]
+		~dev-python/pywlroots-0.15*[${PYTHON_USEDEP}]
```

```suggestion
		=dev-python/pywlroots-0.15*[${PYTHON_USEDEP}]
```
`~` is only for non-wildcard exact version